src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -28,6 +39,80 @@ export const ConfirmBidRoute: React.FC<BidProps> = props => {
     saleArtwork,
   })
 
+  function createBidderPosition(maxBidAmountCents: number) {
+    return new Promise(async (resolve, reject) => {
+      commitMutation<ConfirmBidCreateBidderPositionMutation>(
+        props.relay.environment,
+        {
+          onCompleted: data => {
+            resolve(data)
+          },
+          onError: error => {
+            reject(error)
+          },
+          mutation: graphql`
+            mutation ConfirmBidCreateBidderPositionMutation(
+              $input: BidderPositionInput!
+            ) {
+              createBidderPosition(input: $input) {
+                result {
+                  status
+                  message_header
+                  message_description_md
+                }
+              }
+            }
+          `,
+          variables: {
+            input: {
+              sale_id: sale.id,
+              artwork_id: artwork.id,
+              max_bid_amount_cents: maxBidAmountCents,
+            },
+          },
+        }
+      )
+    })
+  }
+
+  function handleMutationError(actions: FormikActions<object>, error: Error) {
+    logger.error(error)
+
+    let errorMessages: string[]
+    if (Array.isArray(error)) {
+      errorMessages = error.map(e => e.message)
+    } else if (typeof error === "string") {
+      errorMessages = [error]
+    } else if (error.message) {
+      errorMessages = [error.message]
+    }
```

Comment:
I wonder if we should be this defensive about the type of `error` here. It's only used in one place and Promise's `catch` should only inject an `Error` object.